src/App.js

```diff
@@ -8,7 +8,7 @@ import NotFound from './components/404/NotFound';
 const App = () => (
   <Provider store={store}>
     <Router>
-      <div>
+      <fragment>
```

Comment:
You did not import `Fragment` from React in this file and Fragment should be in uppercase.

Reply:
I've corrected this error, thanks.